tedge/src/cli/disconnect/bridge.rs

```diff
@@ -0,0 +1,107 @@
+use crate::cli::connect::TEDGE_BRIDGE_CONF_DIR_PATH;
+use crate::command::{Command, ExecutionContext};
+use crate::services::{
+    mosquitto::MosquittoService, tedge_mapper::TedgeMapperService, SystemdService,
+};
+use crate::utils::paths;
+use crate::utils::users::{UserManager, ROOT_USER};
+use which::which;
+
+use super::error::DisconnectBridgeError;
+
+#[derive(Debug)]
+pub struct DisconnectBridge {
+    pub(crate) config_file: String,
```

Comment:
This is bad for testing. You are hardcoding the path to the config_file more or less, because you are constructing it below based on a constant. 

It's better to inject the full path to the `config_file` as `FilePath` or `PathBuf` and build the path in `BuildCommand`.